Suko/View Controller/SUKDetailsViewController.m

```diff
@@ -70,29 +71,31 @@ - (NSString *)dropdownMenu:(MKDropdownMenu *)dropdownMenu titleForRow:(NSInteger
 }
 
 - (void)dropdownMenu:(MKDropdownMenu *)dropdownMenu didSelectRow:(NSInteger)row inComponent:(NSInteger)component {
-    NSMutableArray<NSMutableArray *> *currentAllData = [PFUser currentUser][@"list_data"];
+    NSMutableArray<NSMutableArray *> *currentAllData = [PFUser currentUser][kPFUserListDataKey];
     NSNumber *malID = [NSNumber numberWithInt:self.animeToDisplay.malID];
     
     if(row == 0) { // User clicked on "remove from lists"
         for(int i = 0; i < [currentAllData count]; i++) {
-            // row - 1 because row 0 is "Remove from List"
             if([currentAllData[i] containsObject:malID]) {
                 [currentAllData[i] removeObject:malID];
                 break;
             }
         }
+        
+        [PFUser currentUser][kPFUserListDataKey] = currentAllData;
+        [[PFUser currentUser] saveInBackground];
         [self.dropdownMenu closeAllComponentsAnimated:YES];
```

Comment:
Since these lines are called in both branches, can move them outside of the branch to deduplicate the code.